### PR TITLE
fix show title

### LINKED ISSUE #683: Unicode miscellaneous and punctuation characters missing
Looks like if a character isn't a part of the custom font, then it shows up empty 
i.e. "╚𝒾𝓈𝒶╔𝓇𝑒𝒶𝓀♦¶DXDF" => " 𝒾𝓈𝒶 𝓇𝑒𝒶𝓀  DXDF"
It shows up correctly as the tab's title and as it's own entry in the schedule
scheduled show on datafruits that isn't display a dj's name correctly
I'll make a PR for changing the font used on this page to cursive to emulate what we had before, but with more character support

### PR DESCRIPTION
**Pull request by dudethatbe**
fixes #683

### CODE REVIEW

File: app/templates/home/show.hbs [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
+  <div style="background-image: url({{model.thumbImageUrl}})" alt={{model.title}} class="w-full h-full show-image">
+  </div>
   {{else if model.host.imageMediumUrl}}
-    <div style="background-image: url({{model.host.imageMediumUrl}})" alt={{model.dj.username}} class="w-full h-full show-image"></div>
+  <div style="background-image: url({{model.host.imageMediumUrl}})" alt={{model.dj.username}}
+    class="w-full h-full show-image"></div>
   {{/if}}
   <div class="bg-df-pink show-info">
-    <h1 class="font-debussy text-3xl">
+    <h1 class="cursive text-3xl">
```
Comment (mcfiredrill) [inline app/templates/home/show.hbs:11]: Is the cursive class a tailwind built in one ?

Comment (dudethatbe) [reply app/templates/home/show.hbs:11]: oh shoot it isn't. I think I have to add it to the config https://tailwindcss.com/docs/font-family#customizing

Comment (mcfiredrill) [reply app/templates/home/show.hbs:11]: @dudethatbe Add a `cursive` entry to the object here just like the others, and you should be able to use the `font-cursive` class everywhere.
https://github.com/datafruits/datafruits/blob/master/app/tailwind/config.js#L7

**Comment by mcfiredrill** [comment]
Looks great @dudethatbe , can you merge master to fix the conflict?

**Comment by mcfiredrill** [comment]
I fixed it on the web interface. @dudethatbe

**Pull request merged**